src/SBInclinedExponential.cpp

```diff
@@ -220,15 +220,11 @@ namespace galsim {
 
     double SBInclinedExponential::SBInclinedExponentialImpl::maxSB() const
     {
-        // When the disk is face on, the max SB is flux / 2 pi r0^2
-        // When the disk is edge on, the max SB is flux / 2 pi r0^2 * (r0/h0)
-        double maxsb = _flux * _inv_r0 * _inv_r0 / (2. * M_PI);
-        // The relationship for inclinations in between these is not linear.
-        // Empirically, it is vaguely linearish in sqrt(cosi), so we use that for
-        // the interpolation.  It's accurate to ~10-20% for moderate values of h0/r0.
-        double sc = sqrt(std::abs(_cosi));
-        maxsb *= (_h0 * sc + _r0 * (1.-sc)) / _h0;
-        return std::abs(maxsb);
+        // Maximum surface brightness isn't analytic, and is only needed for photon-shooting,
+        // so this is disabled.
+        throw std::runtime_error(
+            "The maxSB method of SBInclinedExponential is not yet implemented");
+        return 0;
```

Comment:
Either way.  I'd probably be more inclined to leave off the extra factor for simplicity sake.  A factor of 2 isn't really a big deal for how this function is used and these are better than that already.  But OK if you'd prefer to leave them in.